tests/test_freshdesk_pagination.py

```diff
@@ -43,7 +43,7 @@ def test_run(self):
             with self.subTest(stream=stream):
                 # Not able to generate more data as roles stream requires pro account.
                 # So, updating page_sie according to data available.
-                if stream == "roles":
+                if stream in ["roles", "ticket_fields"]:
                     page_size = 2
```

Comment:
Where are we setting `page_size=2` in the test? I could not find it in get_properties()

Reply:
We are setting `page_size`  property in the `base.py`. It was added in the [PR#51](https://github.com/singer-io/tap-freshdesk/pull/51) and we have raised this pr from it. Back merge remained and that's why this change was not reflected here.